CMakeLists.txt

```diff
@@ -37,6 +37,14 @@ if (NOT CMAKE_SYSTEM_NAME)
 	message(FATAL_ERROR "No target OS set")
 endif()
 
+# Add an option to exclude rdiff executable from build
+# This is useful, because it allows to remove POPT dependency if a user is interested only in the
+# rsync library itself and not in the rdiff executable
+option(BUILD_RDIFF "Whether or not to build rdiff executable" ON)
+
+# Add an option to include compression support
```

Comment:
Please mention https://github.com/librsync/librsync/issues/8 here, that it's not actually functional